pyproject.toml

```diff
@@ -51,5 +51,5 @@ venvPath = ".pixi/envs"
 venv = "py312"
 
 [tool.mypy]
-python_version = "3.8"
+python_version = "3.9"
```

Comment:
The CI already had the test for py38 removed, so I assume it's fine to increment this?